### ISSUE TITLE
[macOS] Format Error

### ISSUE THREAD

**Issue by hexzerodev** [Galarius/vscode-opencl#49]
I'm getting this error with the command `Format Document` on a `*.cl` file (and also when saving with `Format on Save` enabled)

```
[OpenCL Formatter] Failed to find default 'clang-format' binary in 'ms-vscode.cpptools' extension.
```

Any guesses to what the issue is? 

`ms-vscode.cpptools` installed
VSCode Version: 1.76.0
MacOS 12.3.1 
`OpenCL: Info` works

**Comment by Galarius** [comment]
Hello, @hexzerodev! Thank you for reporting this. The executable name in `ms-vscode.cpptools` extension has changed. I will make an adjustment in the upcoming `vscode-opencl` release. 

As a workaround, please specify the absolute `clang-format` path in the VS Code settings:

```json
{
    "OpenCL.formatting.name": "/Users/<user name>/.vscode/extensions/ms-vscode.cpptools-<version>-darwin-x64/LLVM/bin/clang-format"
}
```

*Replace `<user name>` and `<version>` with the appropriate values.*

**Comment by geeshta** [comment]
Hi, I have a problem when running on Ubuntu, I've tried setting the absolute path to the formatter:
```
"OpenCL.formatting.name": "/home/<my username>/.vscode/extensions/ms-vscode.cpptools-1.14.5-linux-x64/LLVM/bin/clang-format"
```
However whenever I try to format a .cl file, I get this popup:
```
[OpenCL Formatter] Failed to format document. Check the console in dev tools to find errors when formatting.
```
And when I go to Output -> Extension Host I see
```
2023-03-31 12:21:28.793 [error] [galarius.vscode-opencl] provider FAILED
2023-03-31 12:21:28.793 [error] undefined
```

What works for me is installing clang-format directly with `sudo apt install clang-format` and then setting the path to its binary 
```
"OpenCL.formatting.name": "/usr/bin/clang-format"
```

**Comment by Galarius** [comment]
@geeshta, thank you for reporting, I'll take a look.

**Comment by Galarius** [comment]
Fixed in [0.8.0](https://github.com/Galarius/vscode-opencl/releases/tag/0.8.0) for macOS.
